lib/ui/components/qrstatuswidget.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:qrapp/models/seed.dart';
+import 'package:qrapp/models/seedstatus.dart';
+import 'package:intl/intl.dart';
+
+class QRStatusWidget extends StatefulWidget {
+  SeedStatus status;
+  SeedModel? seed;
+  QRStatusWidget({Key? key, required this.status, required this.seed})
+      : super(key: key);
+
+  @override
+  _QRStatusWidgetState createState() => _QRStatusWidgetState();
+}
+
+class _QRStatusWidgetState extends State<QRStatusWidget> {
+  @override
+  Widget build(BuildContext context) {
```

Comment:
**question**: why aren't you using a `Consumer` to lister for changes on your `ChangeNotifierProvider`? Do you think the current approach works better?